addons/website/static/src/js/backend/dashboard.js

```diff
@@ -362,9 +364,18 @@ var Dashboard = AbstractAction.extend({
         } else if (this.date_range === 'year') {
             additional_context = {search_default_year: true};
         }
-        this.do_action($action.attr('name'), {
-            additional_context: additional_context,
-            on_reverse_breadcrumb: this.on_reverse_breadcrumb
+        this._rpc({
+            route: '/web/action/load',
+            params: {
+                'action_id': $action.attr('name'),
+            },
+        })
+        .then(function (r) {
+            r.domain = pyUtils.assembleDomains([r.domain, `[('website_id', '=', ${self.website_id})]`])
+            return self.do_action(r, {
+                'additional_context': additional_context,
+                'on_reverse_breadcrumb': self.on_reverse_breadcrumb
```

Comment:
no need to quote keys in js

Reply:
I added them on purpose to avoid the linter errors, since I changed the line.
Or maybe the linter rules changed and this one is not considered as an error anymore?

Reply:
I don't know which linter you use, but this should be fine without quotes (see what you did 5 lines above)

Reply:
I mean the issue on my linter is that it is not in camel case, not really the fact is has/has no quotes

Reply:
I am missing `"camelcase": ["warn", {"properties": "never"}],` it seems, nervermind